server/channels/api4/content_flagging_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package api4
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetFlaggingConfiguration(t *testing.T) {
+	mainHelper.Parallel(t)
+	if *mainHelper.GetSQLSettings().DriverName == model.DatabaseDriverMysql {
+		t.Skip("Content flagging tests are not supported on MySQL")
+	}
```

Comment:
This should be removed if https://github.com/mattermost/mattermost/pull/33458 goes in first